docs/readmes/lte/sessiond.md

```diff
@@ -1,3 +1,9 @@
+---
+id: sessiond
+title: Sessiond Management in Magma
```

Comment:
```suggestion
title: Session Management in Magma
```

Reply:
A suggestion to make the title in line with the title of the doc

Reply:
Of course, pushed